src/App.js

```diff
@@ -1,7 +1,9 @@
 import React, { useState } from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Routes from './routes';
 import './App.scss';
 import data from './data/dummy.json';
-import { Button, Form, Container, Row, Jumbotron } from 'react-bootstrap'
+import { Button, Form, Container, Jumbotron } from 'react-bootstrap';
 
 const App = () => {
   const [userAnswers, setUserAnswers] = useState({});
```

Comment:
I would've expected the quiz questions to be moved to the `Quiz` component so that when I load up the app I only see the `Router` routes, then I can visit `/quiz` to see and answer the questions. Maybe we can move them as part of this pull request?

Reply:
done